### ISSUE TITLE
tbx_add_to_collection.py needs to detect when it can't insert because of a duplicate record

### ISSUE THREAD

**Issue by nickrsan** [ucd-cws/PISCES#19]
**[Original report](https://bitbucket.org/nickrsan/pisces/issue/19) by Nicholas Santos (Bitbucket: [nickrsan](https://bitbucket.org/nickrsan), GitHub: [nickrsan](https://github.com/nickrsan)).**

----------------------------------------

from the commit:tbx_add_to_collection.py where it won't crash if a record already exists
for that observation in the collection - it'll just print a notice and continue
- still needs to be fixed so that it detects if it's a relationship problem and prints nothing
and prints an error if it's not.

Basically, if it's going to create a duplicate record (bad), we don't want it to and it won't. But we want it to detect in the except block that that's WHY they query failed and print an error message if it's not (so we know that the query couldn't be run for another reason!)
